bin/run-docker

```diff
@@ -4,16 +4,40 @@ set -e
 
 uname_S=$(uname -s 2>/dev/null || echo not)
 
+
+
+
+
 ARDUINO_LOCAL_LIB_PATH="${ARDUINO_LOCAL_LIB_PATH:-${HOME}/Arduino}"
 
 if [ "${uname_S}" = "Darwin" ]; then
     ARDUINO_LOCAL_LIB_PATH="${ARDUINO_LOCAL_LIB_PATH:-${HOME}/Documents/Arduino}"
+    # This stops macos from copying resource forks into thigns like tar
+    export COPYFILE_DISABLE=true
 fi
 
 BOARD_HARDWARE_PATH="${BOARD_HARDWARE_PATH:-${ARDUINO_LOCAL_LIB_PATH}/hardware}"
 
-docker build -t kaleidoscope/docker etc
+
+XFER_DIR="$(pwd)/.docker_xfer"
+
+mkdir -p "${XFER_DIR}"
+
```

Comment:
We _should_ be emitting a status message here, I believe. Tarring things up can take a while if there's all kinds of random stuff in any of those dirs, leaving the user wondering what exactly the script is doing.

Reply:
I don't mind the status message, but ... it takes a lot of stuff to make that tarball slow ;)

Reply:
Yeah, I know. I had a coredump in there for some reason, and it took 30 seconds for the tarballs alone.